### PR TITLE
feat(0.2): adds psr-4 autoloading in functions

### PR DESCRIPTION
**Pull request by nunomaduro**
This pull request prepares the Laravel Plugin for Pest 0.2

It gives access the full Laravel testing api allowing to perform stuff like this:

```php
<?php

use Pest\Laravel\{get};

get('/')->assertSee('laravel');
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,12 +14,23 @@
     "require": {
         "php": "^7.3",
         "laravel/framework": "^7.0",
-        "pestphp/pest": "^0.1.2"
+        "pestphp/pest": "@dev"
     },
     "autoload": {
-        "files": ["src/Autoload.php"]
+        "psr-4": {
+            "Pest\\Laravel\\": "src/"
```
Comment (owenvoke) [inline composer.json:21]: Should this be `Pest\\PluginLaravel` like the other plugins, or are plugins moving to not use "Plugin" in the namespace?

If so, should the Faker and Init plugins should be moved to `Pest\Faker` and `Pest\Init` respectively?

Comment (nunomaduro) [reply composer.json:21]: I was thinking about moving ( sorry ) again to `Pest\Faker`, and `Pest\Laravel`. What do you think?

Comment (owenvoke) [reply composer.json:21]: No need to be sorry, sounds good to me. :+1: I'll have a look at updating those repos tomorrow maybe?

Comment (nunomaduro) [reply composer.json:21]: Thanks!

File: src/Authentication.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+use Illuminate\Foundation\Testing\TestCase;
+
+/**
+ * Set the currently logged in user for the application.
+ *
+ * @return TestCase
+ */
+function actingAs(Authenticatable $user, string $driver = null)
+{
+    return test()->actingAs(...func_get_args());
```
Comment (fetzi) [inline src/Authentication.php:17]: Why aren’t you using $user & $driver? 🤔 Also relevant for the methods below.

Comment (nunomaduro) [reply src/Authentication.php:17]: To avoid mistakes, I am *always* using this approach `...func_get_args()`.

**Pull request merged**